pal_event_server/lib/src/allowlist_default_rules.dart

```diff
@@ -64,12 +67,14 @@ List<AllowListRule> createRules() {
   rules.add(AllowListRule.ofAppUsed(r'Brave'));
   rules.add(AllowListRule.ofAppContent(r'Terminal', r'.*'));
   rules.add(AllowListRule.ofAppContent(r'Taqo', r'.*'));
-  rules.add(AllowListRule.ofAppContent(r'.*', r'.*search.*'));
-  rules.add(AllowListRule.ofAppContent(r'.*', 'Mail'));
-  rules.add(AllowListRule.ofAppContent(r'.*', 'Calendar'));
+  rules.add(AllowListRule.ofAppContent(r'.*', 'Google.com - Mail'));
+  rules.add(AllowListRule.ofAppContent(r'.*', 'Google.com - Calendar'));
```

Comment:
I think these rules are applied after `wipeDetailsOnEvent`, so that app content already becomes `Mail` and `Calendar`.

Reply:
Oops. Yes. Fixed.